kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java

```diff
@@ -96,14 +96,14 @@ protected void addExtraCustomClasses(Map<String, Class<?>> extraClasses)
                                        kieContainer.getClassLoader()));
     }
 
-    @Test
```

Comment:
Please uncomment the tests here.

Reply:
uncommented, damn it I did it again ;) thanks for catching it